tests/installation/select_visible_patterns_from_bottom.pm

```diff
@@ -0,0 +1,30 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Select a pattern from the bottom
+#    You can pass
+#    PATTERNS=yast-development
+#
+# Maintainer: QA SLE YaST team <qa-sle-yast@suse.de>
+
+use base 'y2_installbase';
+use strict;
+use warnings;
+use testapi;
+
+sub run {
+    my ($self) = @_;
+    $self->go_to_patterns();
+    my @patterns = grep($_, split(/,/, get_required_var('PATTERNS')));
+    # delete special 'default' key from the check
+    @patterns = grep { $_ ne 'default' } @patterns;
```

Comment:
we don't need this line, we should remove it from settings via MR.

Reply:
Technically yes, however someone checking the settings can interpret that "PATTERNS=yast_develpoment" may mean *only* yast development, and not default patterns, no ?

Reply:
our way is to check the name of the test module, if we pass as settings or test data is just by our own convenience, we have `tests/installation/select_only_visible_patterns_from_top.pm` for disambiguation` with `only` in the name.